### ISSUE TITLE
Inline comments not recognized ( *> comments)

### ISSUE THREAD

**Issue by Reinhard-Prehofer** [uwol/cobol85parser#13]
Inline-comments in Cobol can be used since Cobol-2002 (extending the ANSI-85 norm)
*> starts a cobol-comment till end-of-line, the same like // for Java.

Parsing leads to error:
line 21:30 mismatched input '*' expecting <EOF>
and line 21 is the one withe the *> ANSI End-of-Line Comment .

Note: I am aware of the Parser being named 85 ;)

```
 01  Student.
   03  nachname    pic x(20).
   03  vorname     pic x(20).
   03  geschlecht  pic x(1).  *> ANSI End-of-Line Comment 
     88  mann      values 'M' 'm'. 
	 88  frau      value 'F', 'f', 'W', 'w'.
	 88  egal      values 'a' thru 'z',
	      'A' thru 'Z'.
```

**Comment by uwol** [comment]
Already fixed in 4c80d4e540f6277822600c43846ba827c5e047b7. Added unit test in 888899fdb9410631386314cb11017c72446ae738.

**Comment by albertolovato** [comment]
Inline comment at start of line without space still gives error, as in

`line 4:11 mismatched input '*>' expecting <EOF>`


       PROCEDURE DIVISION.
           *>DISPLAY "Hello World".
           DISPLAY "Hello World".

**Comment by uwol** [comment]
Ah, you are right. Thanks for issuing! Fixed in e25c264f911e2ee659ad010fa312a90d8807cbe7 and added unit tests.

The fix was a little bit tricky, so if I missed something, please let me know.
